mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/validator/IdentitySchemaValidator.java

```diff
@@ -0,0 +1,97 @@
+package io.mosip.esignet.mock.identitysystem.validator;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.networknt.schema.*;
+import io.mosip.esignet.mock.identitysystem.dto.IdentityData;
+import io.mosip.esignet.mock.identitysystem.dto.RequestWrapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.stereotype.Component;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Set;
+
+@Component
+@Slf4j
+public class IdentitySchemaValidator implements ConstraintValidator<IdentitySchema, Object> {
+
+    @Value("${mosip.mock.ida.identity.create.schema.url}")
+    private String createSchemaUrl;
+
+    @Value("#{${mosip.mock.ida.identity.update.exempted.field}}")
+    private Set<String> exemptedField;
```

Comment:
Rename the key -> mosip.mock.ida.update-identity.non-mandatory.fields
and the exemptedField to nonMandatoryFieldsOnUpdate